Sources/XcodeProj/Scheme/XCScheme+BuildAction.swift

```diff
@@ -70,12 +70,44 @@ extension XCScheme {
             }
         }
 
+        public enum Architectures {
+            case matchRunDestination
+            case universal
+            case useTargetSettings
+
+            fileprivate var xmlString: String? {
+                switch self {
+                case .matchRunDestination:
+                    "Automatic"
+                case .universal:
+                    "All"
+                case .useTargetSettings:
+                    nil
+                }
+            }
+
+            /// Creates a new instance from the given xml string.
+            ///
+            /// For undefined value, initialized as `matchRunDestination`.
+            fileprivate init(_ xmlString: String) {
+                switch xmlString {
+                case "Automatic":
+                    self = .matchRunDestination
+                case "All":
+                    self = .universal
+                default:
+                    self = .matchRunDestination
```

Comment:
From testing in a Xcode, unspecified or random values default to `useTargetSettings`

```suggestion
                    self = .useTargetSettings
```

Alternatively can make this an failable init (`init?`)

Reply:
That's right. It’s my mistake. 🙏